arrow-udf-wasm/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+
+- Fix complilation of Rust UDF using primitive array data types.
+
```

Comment:
The changelog should be put in arrow-udf-macros?

Reply:
Seems that `arrow-udf-macros` is not directly used?